### DISCUSSION TITLE
更新失败怎么回事啊

### DISCUSSION THREAD

**Discussion by muccx** [ywmoyue/biliuwp-lite#663] [Q&A]
### Bug 描述

![PixPin_2024-06-08_14-35-52](https://github.com/ywmoyue/biliuwp-lite/assets/80401688/b2fd036c-6112-47cc-8864-a17c8f80cc81)

### 操作步骤

```plain text
无
```

### 期望结果

无

### 应用来源

Github/其他Github镜像站

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
请查看压缩包中的安装教程，使用脚本安装方式，

**Reply by ywmoyue** [reply]
@muccx

**Comment by ywmoyue** [comment]
@muccx 问题是否解决？
